polyfills/Element/prototype/childElementCount/config.json

```diff
@@ -0,0 +1,12 @@
+{
+  "browsers": {
+    "firefox": "<3.5",
+    "ie": "6 - 8",
+    "opera": "<10.0",
+    "safari": "<4"
```

Comment:
The browsers didn't list here support the feature. They don't need the polyfill. In this case, what should I write? 

```javascript
"chrome": "*",
```
means all versions of chrome need the polyfill, right?